### DISCUSSION TITLE
Wiki

### DISCUSSION THREAD

**Discussion by hrondonuwu** [SingularityLabs-ai/beyond-bard-mini#6] [Q&A]
I've been using the beyondbard extension and noticed it doesn't work for wikipedia. Is there a reason for this? Did I do something wrong? If possible could you make beyondbard support wikipedia?

**Comment by ishandutta2007** [comment]
@NarutoUchiha69420 thanks for pointing out, another user said that too, wikipedia is in our action list, can you move this discussions to [issues](https://github.com/SingularityLabs-ai/beyond-bard-mini/issues) for better tracking

**Reply by divyaprakashdp** [reply]
@ishandutta2007 Created an issue for this - https://github.com/SingularityLabs-ai/beyond-bard-mini/issues/9
